blog/content/post/2023-06-23-Blogpost_yocto_kas.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Blog post - odświeżenie "Minimal Image, Fastest Rpi, Quick Tutorial"
+abstract: 'So far, to build even a minimal image on th raspberry pi the downloading of sources and then configuration had to be done by hand.
+	  Instead Yocto kas is using a project configuration file and does the download and configuration phase.
+	  Kas tool provides an easy mechanism to setup bitbake based projects.
+          Kas makes the setup of a Yocto build environment super simple and super fast.'
+cover: /covers/Cover_Yocto_kas_RPI4.png
+author: ewa.kujawska
+layout: post
+published: true
+date: 2023-06-23
+archives: "2023"
+
+tags:
+  - yocto
+  - poky
+  - kas
+  - rpi
+  - linux
+  - meta-raspberrypi
+  - minimal_image
+  - u-boot
+  - open-source
+  - mickledore
```

Comment:
@Kewkaa, very rarely should we invent new tags. Please check [here](https://blog.3mdeb.com/tags/) what tags we use.